kubernetes/adapter.go

```diff
@@ -65,6 +68,7 @@ type Ingress struct {
 	SSLPolicy        string
 	IPAddressType    string
 	LoadBalancerType string
+	typ              string // "ingress" or "routegroup" to patch status based on that
```

Comment:
maybe call it `resourceType` instead of this weird word? :)

Reply:
fixed

Reply:
I changed the name from typ to resourceType.